### PR TITLE
Configure gulpfile for work with Transifex

### PR DESCRIPTION
**Pull request by SounD120**
#820
Enhanced work with external libraries in `gulpfile.js` by changing their objects types from var to const

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,29 +1,54 @@
...
+const install = require("gulp-install");
+const runSequence = require("run-sequence");
+const ts = require("gulp-typescript");
+const mocha = require("gulp-mocha");
+const GulpExtras = require("./tools/gulp-extras");
+const minimist = require("minimist");
+const os = require("os");
+const fs = require("fs");
+const Q = require("q");
+const es = require('event-stream');
```
Comment (ruslan-bikkinin) [inline gulpfile.js]: @SounD120 Seems to be, you forgot to add `event-stream` to devDependecies.

File: gulpfile.js [hunk 2]
```diff
@@ -232,4 +273,37 @@ gulp.task("release", ["build"], function () {
                 fs.writeFileSync(path.join(__dirname, fileName), fs.readFileSync(path.join(backupFolder, fileName)));
             });
         });
-});
\ No newline at end of file
+});
+
+gulp.task('add-i18n', function () {
```
Comment (ruslan-bikkinin) [inline gulpfile.js]: @SounD120 Could you please add comments for each of the new gulp tasks with short description of what it does?

File: gulpfile.js (hunk 1 above)
Comment (SounD120) [reply gulpfile.js]: Thanks for pointing out. I'll fix that.

File: gulpfile.js (hunk 2 above)
Comment (SounD120) [reply gulpfile.js]: Yes, sure

**Pull request merged**